src/njoy21/input/DTFR/Card2/Iedit.hpp

```diff
@@ -0,0 +1,18 @@
+struct Iedit {
+  using Value_t = int;
+  static std::string name(){ return "iedit"; }
+
+  static std::string description(){
+    return
+      "The iedit value is for edit control, and can either equal 0 or 1.\n"
+      "A value of 0 correspond to table-edits option, while a value of 1\n"
```

Comment:
You mention what is meant by iedit=0, but don't state what is meant by iedit=1. A little more detail should be included.